pkg/promunifi/uap.go

```diff
@@ -321,16 +319,15 @@ func (u *promUnifi) exportRadtable(r report, labels []string, rt unifi.RadioTabl
 				{u.UAP.RadioTxPower, prometheus.GaugeValue, t.TxPower, labelR},
 				{u.UAP.RadioAstBeXmit, prometheus.GaugeValue, t.AstBeXmit, labelR},
 				{u.UAP.RadioChannel, prometheus.GaugeValue, t.Channel, labelR},
-				{u.UAP.RadioCuSelfRx, prometheus.GaugeValue, t.CuSelfRx, labelR},
-				{u.UAP.RadioCuSelfTx, prometheus.GaugeValue, t.CuSelfTx, labelR},
-				{u.UAP.RadioCuTotal, prometheus.GaugeValue, t.CuTotal, labelR},
```

Comment:
Please fix the conflicts, I probably added more things with `_percent` too. Is `_ratio` preferred for percentage gauges?

Reply:
never mind, I'm impatient and want to get all this stuff "released" - I guess giving it time to be better is good too! Really appreciate your help!!